File: services/GroupService/src/main/java/com/se1853_jv/dto/response/NoteResponse.java
package com.se1853_jv.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class NoteResponse {
    private String id;
    private String content;
    private Long coordinationX;
    private Long coordinationY;
    private Integer pageNumber;
    private UUID paperId;
    private UUID collectionId;
    private UUID userId;
}

File: services/paper-service/src/main/kotlin/com/se1853_jv/dto/response/NoteResponse.kt
package com.se1853_jv.dto.response

import java.util.UUID

data class NoteResponse(
    val id: String,
    val content: String,
    val coordinationX: Long,
    val coordinationY: Long,
    val pageNumber: Int,
    val paperId: UUID,
    val collectionId: UUID,
    val userId: UUID
)







